faield on demos/demo_ivfpq_indexing

# Summary
Failed on demos/demo_ivfpq_indexing
make   demos/demo_ivfpq_indexing
g++ -std=c++11 -fPIC -fopenmp -m64 -Wno-sign-compare -g -O3 -Wall -Wextra -DFINTEGER=int  -fopenmp   demos/demo_ivfpq_indexing.cpp   -o demos/demo_ivfpq_indexing
demos/demo_ivfpq_indexing.cpp: In function ‘int main()’:
demos/demo_ivfpq_indexing.cpp:100:39: error: ‘struct faiss::IndexIVFPQ’ has no member named ‘imbalance_factor’
                 elapsed() - t0, index.imbalance_factor ());
                                       ^
<builtin>: recipe for target 'demos/demo_ivfpq_indexing' failed
make: *** [demos/demo_ivfpq_indexing] Error 1


OS: <Ubuntu 16.04 >

Faiss version:  git commit afe0fdc161391b5379ead6204b16035ea7634284 


Running on:
- [ √] CPU
- [ ] GPU

Interface: 
- [ ] C++
- [ √] Python

# Reproduction instructions
cd faiss && make 
make demos/demo_ivfpq_indexing 

Here is my makefile.inc
===============================

CXX          = g++ -std=c++11
CXXCPP       = g++ -std=c++11 -E
CPPFLAGS     = -DFINTEGER=int
CXXFLAGS     = -fPIC -fopenmp -m64 -Wno-sign-compare -g -O3 -Wall -Wextra
CPUFLAGS     = -msse4 -mpopcnt
LDFLAGS      = -fopenmp
LIBS         = -lopenblas
PYTHONCFLAGS = -I/usr/include/python2.7 -I/usr/include/x86_64-linux-gnu/python2.7 -I/usr/lib/python2.7/dist-packages/numpy/core/include

NVCC         = /usr/local/cuda/bin/nvcc
NVCCLDFLAGS  = -L/usr/local/cuda/lib64
NVCCLIBS     = -lcudart -lcublas -lcuda
CUDAROOT     = /usr/local/cuda
CUDACFLAGS   = -I/usr/local/cuda/include
NVCCFLAGS    = -I $(CUDAROOT)/targets/x86_64-linux/include/ \
-Xcompiler -fPIC \
-Xcudafe --diag_suppress=unrecognized_attribute \
-gencode arch=compute_35,code="compute_35" \
-gencode arch=compute_52,code="compute_52" \
-gencode arch=compute_60,code="compute_60" \
-gencode arch=compute_61,code="compute_61" \
-lineinfo \
-ccbin $(CXX) -DFAISS_USE_FLOAT16

OS = $(shell uname -s)

SHAREDEXT   = so
SHAREDFLAGS = -shared

ifeq ($(OS),Darwin)
	SHAREDEXT   = dylib
	SHAREDFLAGS = -dynamiclib -undefined dynamic_lookup
endif

MKDIR_P      = /bin/mkdir -p
PYTHON       = python
SWIG         = swig

prefix      ?= /usr/local
exec_prefix ?= ${prefix}
libdir       = ${exec_prefix}/lib
includedir   = ${prefix}/include

--- a/demos/demo_ivfpq_indexing.cpp
+++ b/demos/demo_ivfpq_indexing.cpp
@@ -97,7 +97,7 @@ int main ()
         index.add (nb, database.data());
 
         printf ("[%.3f s] imbalance factor: %g\n",
-                elapsed() - t0, index.imbalance_factor ());
+                elapsed() - t0, index.invlists->imbalance_factor());
 
         // remember a few elements from the database as queries
         int i0 = 1234;

